src/main/scala/parsec/CharParsers.scala

```diff
@@ -29,10 +29,9 @@ object HelloCharParsers extends CharParsers {
 
   val (strZ, strCombine) = stringFolder
 
-  val wordDigitLetter: Parser[(String, Char)]
-    = (letters ~ digit).fold(strZ, strCombine) map {
-      case (ls, other) => (ls.toString, other)
-    }
+  val wordDigitLetter: Parser[(String, Char)] = (letters ~ digit).fold(strZ, strCombine) map {
+    case (ls, other) => (ls.toString, other)
+  }
 
```

Comment:
Yes, definitely. I didn't do that, it's scalariform's fault!
  
I'll remove it in a while :+1: